### PR TITLE
Plans: Replace the current plan checkmark with a "Your Plan" banner

### LINKED ISSUE #12436: Plans: use ribbon to display indicator of 'your plan'
Currently the green circular checkmark icon is transparent and with the newly colorized plan icons there are a few circumstances where you can see the plan icon color through the checkmark's negative space. Also the various colors clash with the green checkmark a bit. We also have a green plan icons, causing further clash.

image

Let's instead use the 'popular' ribbon to display an indicator of which plan you have. Additionally let's change the call to action button from 'Your Plan' to 'Manage your plan' to make it actionable and more clear. I made the checkmark in the call to action blue-medium as well to reduce the number of colors on the page. 

screen shot 2017-03-22 at 5 34 13 pm

cc @beaulebens @tyxla @richardmuscat

### PR DESCRIPTION
**Pull request by ryelle**
Fixes #12436, switch the current plan indicator from checkmark to a ribbon & change the button label from "Your Plan" to "Manage your plan"

screen shot 2017-04-26 at 1 51 48 pm

To Test:

1. Check out this PR
2. Visit `/plans/$site`
3. Your current plan should now have a ribbon saying "Your Plan"
4. At >1280px wide, you don't see a checkmark
5. The button on your plan should say "Manage your plan" if you own the purchase, or "View plan" if you're just a member on the site.

### CODE REVIEW

File: client/my-sites/plan-features/actions.jsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ const PlanFeaturesActions = ( {
 		upgradeButton = (
 			<Button className={ classes } href={ manageHref } disabled={ ! manageHref }>
 				<Gridicon size={ 18 } icon="checkmark" />
-				{ canPurchase ? translate( 'Your plan' ) : translate( 'Current plan' ) }
+				{ canPurchase ? translate( 'Manage your plan' ) : translate( 'View plan' ) }
```
Comment (a8ci18n) [inline client/my-sites/plan-features/actions.jsx]: Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'View Plan' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/plans-my-plan-label)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Comment (a8ci18n) [inline client/my-sites/plan-features/actions.jsx]: Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Manage Plan', { context: 'Link to current plan from /plans/'} )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by drw158** [comment]
I'd probably get rid of the checkmark inside the button.

File: client/my-sites/plan-features/actions.jsx [hunk 2]
```diff
@@ -43,7 +43,7 @@ const PlanFeaturesActions = ( {
 		upgradeButton = (
 			<Button className={ classes } href={ manageHref } disabled={ ! manageHref }>
 				<Gridicon size={ 18 } icon="checkmark" />
-				{ canPurchase ? translate( 'Your plan' ) : translate( 'Current plan' ) }
+				{ canPurchase ? translate( 'Manage Plan' ) : translate( 'View Plan' ) }
```
Comment (a8ci18n) [inline client/my-sites/plan-features/actions.jsx]: Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Manage Plan', { context: 'Link to current plan from /plans/'} )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/plans-my-plan-label)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by rickybanister** [comment]
I agree with @drw158 I think—once we have prorated upgrades the checkmark won't be so necessary since the higher priced plans will have primary blue buttons. 

Can we ditch the checkmark in the 'manage your plan' button and then ship this? The banner looks great.

**Comment by ryelle** [comment]
Removed the checkmark 👍

**Comment by rickybanister** [comment]
Looks great!

I'm excited to get pro-rated plan purchases finished so the upgrade buttons for the higher tier plans can be 'primary' buttons. 

Let's merge this.

**Comment by johnHackworth** [comment]
codewise, this looks good! :+1:

**Pull request merged**